testkit/src/main/scala/org/bitcoins/testkit/wallet/BitcoinSWalletTest.scala

```diff
@@ -347,6 +373,47 @@ object BitcoinSWalletTest extends WalletLogger {
     } yield funded
   }
 
+  /** Funds a bitcoin-s wallet with 3 utxos with 1, 2 and 3 bitcoin in the utxos */
+  def fundWallet(wallet: UnlockedWalletApi)(
+      implicit ec: ExecutionContext): Future[FundedWallet] = {
+    //get three addresses
+    val addressesF = Future.sequence(Vector.fill(3) {
+      //this Thread.sleep is needed because of
+      //https://github.com/bitcoin-s/bitcoin-s/issues/1009
```

Comment:
Once this is merged, a note should be made in #1009 to remove the `Thread.sleep`